app/src/main/java/org/dicio/dicio_android/skills/navigate/NavigateOutput.java

```diff
@@ -0,0 +1,45 @@
+package org.dicio.dicio_android.skills.navigate;
+
+import android.content.Intent;
+import android.net.Uri;
+
+import org.dicio.dicio_android.output.graphical.GraphicalOutputUtils;
+import org.dicio.skill.chain.OutputGenerator;
+
+import java.util.Locale;
+
+public class NavigateOutput extends OutputGenerator<NavigateOutput.Data> {
+
+    public static class Data {
+        public boolean failed = false;
+        public String address;
+    }
+
+    @Override
+    public void generate(final Data data) {
+        if (data.failed) {
+            final String message = "Specify where you want to navigate to";
+            ctx().getSpeechOutputDevice().speak(message);
+            ctx().getGraphicalOutputDevice().display(GraphicalOutputUtils.buildSubHeader(
+                    ctx().android(), message));
+            return;
+        }
+
+        final String message = "Navigating to " + data.address;
+        ctx().getSpeechOutputDevice().speak(message);
+        ctx().getGraphicalOutputDevice().display(GraphicalOutputUtils.buildSubHeader(
+                ctx().android(), message));
+
+        final String uriGeoSimple = String.format(Locale.ENGLISH,
+                "geo:0,0?q=%s", data.address);
```

Comment:
This seems to work for some apps, although for StreetComplete it seems like the `0,0` makes the app go to the coordinates `0,0` instead of going to the query given at `?q=`. Is the `0,0` required for geo intents? Could you give some documentation as to where you took this from?

Reply:
`geo:0,0?q=my+street+address` is the generic Android intent to show an address on the map, as far as I can tell. I've never used StreetComplete, so I don't know why it doesn't support it.

There are also other, app-specific intents.

Links:
- https://developer.android.com/guide/components/intents-common#ViewMap
- https://osmand.net/docs/technical/algorithms/osmand-intents/

Reply:
For now I will resolve this as I have provided answers, don't see any further comments, and to facilitate completion of this PR before it gets too stale. Feel free to re-open.